src/views/Statistics.vue

```diff
@@ -1,18 +1,9 @@
 <template>
   <b-card class="maincard">
     <b-card-header>
-      <v-row>
-        <v-col cols="11"> Cluster Statistics </v-col>
-        <v-col cols="1">
-          <v-btn icon
-          color="primary"
-          @click="request_cluster_infos">
-            <v-icon>mdi-cached</v-icon>
-          </v-btn>
-        </v-col>
-      </v-row>
+      Cluster Statistics
     </b-card-header>
-    <RequestSpinner v-if="!is_loaded" />
+    <RequestSpinner v-if="all_statistics.length === 0" />
```

Comment:
Since the life-cycle in Vue already updates the all_statistics, we dont need the extra variable